package.json

```diff
@@ -114,6 +114,12 @@
                     "type": "number",
                     "default": 5000,
                     "description": "Controls the save delay in milliseconds."
+                },
+                "local-history.fileLimit": {
+                    "type": "number",
+                    "default": 30,
+                    "minimum": 1,
+                    "description": "Controls the revision limit for a file."
```

Comment:
Minor: ‘controls the maximum number of saved revisions allowed for a given file’